src/components/Table/TableHeader/TableHeader.tsx

```diff
@@ -1,24 +1,34 @@
-import React, { forwardRef } from "react";
+import React, { forwardRef, useMemo } from "react";
 import { VibeComponent, VibeComponentProps } from "../../../types";
 import styles from "./TableHeader.module.scss";
 import { ITableHeaderCellProps } from "../TableHeaderCell/TableHeaderCell";
 import cx from "classnames";
 import { getTestId } from "../../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../../tests/constants";
+import { RowSizes } from "../Table/TableConsts";
 
 export interface ITableHeaderProps extends VibeComponentProps {
   children?: React.ReactElement<ITableHeaderCellProps> | React.ReactElement<ITableHeaderCellProps>[];
+  size?: RowSizes;
```

Comment:
I don't like the fact we need to set the size both for the table header and for the table row.
Instead, we should have a single prop on the `Table` level, you can call it `rowSize` that will propagate the value to both `TableHeader` and `TableRow` through context.

Reply:
Fixed